BoxOffice/Model/BoxOfficeResult.swift

```diff
@@ -0,0 +1,18 @@
+//
+//  BoxOfficeResult.swift
+//  BoxOffice
+//
+//  Created by karen on 2023/07/26.
+//
+
+struct BoxOfficeResult: Decodable {
+    let boxofficeType: String
```

Comment:
```suggestion
    let boxOfficeType: String
```

p3;

다른 변수와 컨벤션이 달라보여요 👀

Reply:
`boxOfficeType`로 다른 변수들과 동일하게 컨벤션 통일을 해주었습니다.